include/cxx.h

```diff
@@ -34,6 +73,9 @@ class String final {
   String(String &&) noexcept;
   ~String() noexcept;
 
+#ifdef CXXBRIDGE_HAS_STRING_VIEW
+  String(std::string_view sv) : String(sv.data(), sv.length()) {}
```

Comment:
We should add "always inline", e.g. `__attribute__((always_inline))` (gcc/clang) and `__forceinline` (msvc), to make sure it gets inlined even if e.g. people disable inlining, etc.